examples/test_insert_points.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import numpy as np
+
+from meshpy.tet import MeshInfo, Options, build
+
+if __name__ == '__main__':
+    logger = logging.getLogger('test_insert_points.py')
```

Comment:
Could you add a version of this to the (pytest-driven) tests under `test/`, to make sure this doesn't get inadvertently broken?

Reply:
Done?

Reply:
Looks good. Please also take care of the Flake8 linter failures.

Reply:
Got it. Fixed.

Reply:
Bump

Reply:
Didn't you mention in #70 that this PR was no longer needed? Could you explain?

Reply:
Yes. But this appears to be the correct way to implement this and IMO it makes sense to cover tetgen's interface regardless of whether there are parts of it that are a little redundant. Some people looking for a wrapper library for tetgen may be familiar with one approach and not the other.